content/04.results.md

```diff
@@ -73,24 +73,40 @@ Negative values indicate molecular biology concepts, while positive values indic
 ](https://raw.githubusercontent.com/danich1/annorxiver/4e0c90c590dd7958dc424a1b4fefcecccb3c28ad/biorxiv/pca_association_experiment/output/pca_plots/figures/category_box_plot_pc1.svg){#fig:pca1_pointplot}
 
 ![
-On average majority of bioRxiv preprint categories contain more neuroscience concepts than bioinformatic concepts.
+The second PC groups neuroscience related preprint categories and bioinformatics related preprint categories together.
 This is box plot shows preprints in each article category projected along the PC2 direction.
 Negative values indicate neuroscience concepts, while positive values indicate bioinformatic concepts.
 ](https://raw.githubusercontent.com/danich1/annorxiver/4e0c90c590dd7958dc424a1b4fefcecccb3c28ad/biorxiv/pca_association_experiment/output/pca_plots/figures/category_box_plot_pc2.svg){#fig:pca2_pointplot}
 
+| Title [citation]                                                                                                                                                  | PC_1              | License      | 
```

Comment:
Wow - I'm amazed at how restrictive the licenses are. Can we link to figure 1 for each of them instead of embedding them?